sw/airborne/modules/computer_vision/lib/v4l/v4l2.c

```diff
@@ -92,8 +97,8 @@ static void *v4l2_capture_thread(void *data)
     assert(buf.index < dev->buffers_cnt);
 
     // Copy the timestamp
-    memcpy(&dev->buffers[buf.index].timestamp, &buf.timestamp, sizeof(struct timeval));
-
+    dev->buffers[buf.index].timestamp = buf.timestamp.tv_usec;
```

Comment:
Mmm, I think I agree with you. @fvantienen , what do you think? At first I thought you wanted to save tv_sec*1000000+tv_usec, but this gives us a flight time of 4294 seconds, just over an hour, before it overflows. 
Shall we just keep the timestamp struct?